Complete & Mobilize Questions Section

victoriazzz opened the pull request:
Added "Who are ULAs" Info Button Header and "Meet the ULAs" Info Button, and mobilized for mobile use. There is a small issue when decreasing the window size where the "What courses can I get help with?" Info Button Header wraps onto a different line and putting the "Browse Courses" Button onto a different line, making the section look a bit uneven (as shown in the last screenshot). Any advice would be appreciated on how to fix this.

Fixes Issue #87 
<img width="1552" height="908" alt="Screenshot 2025-08-03 at 9 44 36 AM" src="https://github.com/user-attachments/assets/0259b0f2-62e5-453b-921b-1d7ef1bedde0" />
<img width="686" height="908" alt="Screenshot 2025-08-03 at 9 44 53 AM" src="https://github.com/user-attachments/assets/6978c1c3-658c-4ff8-9453-893ab608ace8" />
<img width="1097" height="908" alt="Screenshot 2025-08-03 at 9 45 13 AM" src="https://github.com/user-attachments/assets/e90772c3-f33e-46f2-83cf-9a9099c2e57c" />

ALE00111 commented:
Is the 3rd screenshot a set size you found within the inspect > mobile options?

ALE00111 commented on src/components/home/InfoButton.tsx:
```diff
@@ -12,10 +12,10 @@ interface infoButtonProps {
     <Link href={link}>
-      <div className="m-4 flex w-2/5 items-center justify-between px-6 py-5 shadow-lg">
-        <div className="flex items-center gap-3">
+      <div className="m-4 flex min-w-[380px] gap-8 px-6 py-5 shadow-lg">
```
Avoid using min width and kinda stay away from exact pixel values like this. I suggest trying a specific width for scenarios like this for mobile cases. But for this example, I can see that the min width isn't really needed here and it might be messing up sizes on other screens.

ALE00111 replied:
Try replacing the gap with justify-evenly and adjusting the padding values. It'll make the button more consistent. Also a general rule of thumb is if we're using flex, we want to utilize other flex commands with it. Ex: flex justify-center.

ALE00111 commented on src/components/home/Questions.tsx:
```diff
@@ -1,12 +1,19 @@
-      <InfoButtonHeader text="What courses can I get help with?" />
-      <InfoButton icon={blueBook} text="Browse Courses" link="/about" />
+    <div className="flex flex-col justify-evenly px-2 pt-8 md:flex-row">
+      <div className="min-w-[392px] md:w-2/5">
```
Same comment as above for min width

ALE00111 commented on src/components/home/Questions.tsx:
```diff
@@ -1,12 +1,19 @@
+        <InfoButtonHeader text="What courses can I get help with?" />
+        <InfoButton icon={blueBook} text="Browse Courses" link="/about" />
+      </div>
+      <div className="min-w-[392px] md:w-2/5">
```
Same comment as above for min width

ALE00111 reviewed (changes requested):
a couple comments, but looking good so far!

victoriazzz commented:
In the updates, I removed any min-w commands. I also fixed up some justifying and margin spacing. Finally, I edited the Info Button's arrow size to shrink so it won't merge with the text anymore.
<img width="767" height="908" alt="Screenshot 2025-08-10 at 12 47 21 PM" src="https://github.com/user-attachments/assets/fb0b3d2b-0210-40fb-821a-bf5185714cf6" />

ALE00111 commented on src/components/home/Questions.tsx:
```diff
@@ -1,12 +1,19 @@
-    <div className="">
-      <InfoButtonHeader text="What courses can I get help with?" />
-      <InfoButton icon={blueBook} text="Browse Courses" link="/about" />
+    <div className="flex flex-col justify-evenly px-2 pt-8 md:flex-row">
```
a fix for your comment would be to change md:flex-row to xl:flex-row, since we can just have them stack on top of each other if it's a non-desktop screen, and also can you add some padding on the sides as well.

ALE00111 commented on src/components/home/Questions.tsx:
```diff
@@ -1,12 +1,19 @@
-      <InfoButtonHeader text="What courses can I get help with?" />
-      <InfoButton icon={blueBook} text="Browse Courses" link="/about" />
+    <div className="flex flex-col justify-evenly px-2 pt-8 md:flex-row">
+      <div className="md:w-2/5">
```
change this to xl as well

ALE00111 commented on src/components/home/Questions.tsx:
```diff
@@ -1,12 +1,19 @@
+        <InfoButtonHeader text="What courses can I get help with?" />
+        <InfoButton icon={blueBook} text="Browse Courses" link="/about" />
+      </div>
+      <div className="md:w-2/5">
```
change this to xl as well

ALE00111 reviewed (changes requested):
slight change!
